packages/cli/src/commands/create-test/create-test.command.ts

```diff
@@ -20,7 +19,20 @@ import {
 
 interface Options
   extends Omit<RecordCommandHandlerOptions, "devTools" | "bypassCSP">,
-    RawReplayCommandHandlerOptions {}
+    Omit<
+      RawReplayCommandHandlerOptions,
+      | "screenshot"
+      | "appUrl"
+      | "simulationIdForAssets"
+      | "maxDurationMs"
+      | "maxEventCount"
+      | "storyboard"
+      | "diffThreshold"
+      | "diffPixelThreshold"
+      | "sessionId"
+      | "save"
+      | "baseSimulationId"
+    > {}
```

Comment:
We have to Omit stuff now, since it forces us to have correct typings. Before we were falsing implying in our types that we had valid numerical values for things like diffThreshold, when they were actually undefined.